src/Accessor.tsx

```diff
@@ -0,0 +1,7 @@
+import { useCustomSelector } from "./redux/store";
+
+export default function Accessor(props: any) {
+  const { authToken } = useCustomSelector((state) => state.appointmentReducer);
+  const { children } = props;
+  return <>{authToken && { ...children }}</>;
```

Comment:
it is not correct, you should not spread children, let's use it next way:
first of all Accessor is not obligated to render children only for authenticated users.

1. so you should add props like authOnly: boolean and depending on that use logic for authentications.